include/Makefile.am

```diff
@@ -17,12 +17,14 @@ noinst_HEADERS = \
 	nccl_ofi_log.h \
 	nccl_ofi_freelist.h \
 	nccl_ofi_param.h \
+	nccl_ofi_tuner.h \
```

Comment:
dont u want to be specific in the name that this is aws_ofi tuner or it is a generic headers that would be used for other plugins ?